### ISSUE TITLE
Documentation is incomplete and inaccurate, which makes it very hard to follow. 

### ISSUE THREAD

**Issue by vinayvenkat** [GoogleCloudPlatform/marketplace-k8s-app-example#117]
Hello,

The scheme file references variables such as "name" which maps to "NAME". But this variable is not setup correctly in the environment file. It references export APP_INSTANCE_NAME=wordpress-1

Secondly, the schema.yaml requires many variables which have not been appropriately filed out in the example exports of the environment variables. While I understand that this is only an example, being incomplete is misleading when users are trying to use the documentation to infer what exactly is required to onboard a market place solution. 

My request is for you to please fix this. 
Thanks.

**Comment by ffoysal** [comment]
moreover inside wordpress folder got error
```
$ make app/doctor

(unset)
(unset)
---- REGISTRY = gcr.io/
---- GCS_URL = gs://
---- NAMESPACE = default
---- MARKETPLACE_TOOLS_TAG = 0.8.0
---- APP_DEPLOYER_IMAGE = gcr.io//example/wordpress/deployer:latest

Building app/doctor...
.build/app/dev doctor
readlink: illegal option -- f
usage: readlink [-n] [file ...]
make: *** [app/doctor] Error 1```
